### PR TITLE
xpath relative pathing issue fix

### PR DESCRIPTION
**Pull request by csoete**
Fixed issue #645 with elementIdElement not using relative xpath.

### CODE REVIEW

File: lib/helpers/find-element-strategy.js [hunk 1]
```diff
@@ -3,8 +3,14 @@ var ErrorHandler = require('../utils/ErrorHandler.js');
...
 
     var value = arguments[0],
+        relative = (arguments.length > 1 ? arguments[1] : false),
         using, query, tag, classOrId, classOrIdName;
 
+    if (relative)
+        xpathPrefix = './';
+    else
+        xpathPrefix = '//';
+
```
Comment (christian-bromann) [inline lib/helpers/find-element-strategy.js]: missing braces

File: test/site/www/table.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+    <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1, maximum-scale=1">
+    <link rel="stylesheet" href="css/bootstrap.css">
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+	<table>
+	    <tr><td>2015-03-01</td><td>12:00</td></tr>
+	    <tr><td>2015-03-02</td><td>12:00</td></tr>
+	</table>
```
Comment (christian-bromann) [inline test/site/www/table.html]: can we move that table to `index.html`? I don't thing we need another site only for these two columns

File: test/spec/issue645.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+var Q = require("q");
+
+describe.only('issue645', function() {
+
+    before(h.setup({url: 'http://localhost:8080/test/site/www/table.html'}));
+
+    it('should select cell using context of row', function() {
+
+        this.timeout(60000);
+        this.slow(5000);
```
Comment (christian-bromann) [inline test/spec/issue645.js]: are these two lines required?

**Comment by christian-bromann** [comment]
couple of comments, additionally:
- could you apply the change you did in `elementIdElement.js` also to `elementIdElements.js`
- here are a couple of jslint errors that need to get fixed

Thanks!

**Comment by csoete** [comment]
Thanks for the feedback, will update the pull request according to your suggestions.

**Comment by christian-bromann** [comment]
Awesome, thanks for the fix! Hope to see more contributions like that :wink: !

**Comment by christian-bromann** [comment]
I will publish this as soon as I've fixed #629

**Pull request merged**